src/pages/Pools/RemoveLiquidity/index.tsx

```diff
@@ -180,11 +180,13 @@ export default function RemoveLiquidity() {
       primaryType: 'Permit',
       message,
     })
+    console.log('hello', message)
```

Comment:
if we need the logs we need to have proper names. So we can know which message is.

For eg.
```suggestion
    console.log('onAttemptToApprove message', message)
```

I'll do the same for all of the logs.

Reply:
Just cleaned all the logs. We don't need them :)